recipes/sciluigi/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sciluigi" %}
+{% set version = "0.9.4b5" %}
+{% set compress_type = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "201f360d48352d0ba582824e25a03e0dd026a613764ecc7cab576541dafc66bb" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - luigi >=1.3,<=2.1
+
+test:
+  imports:
+    - sciluigi
+
+about:
+  home: https://github.com/samuell/sciluigi
+  # license_file: No MANIFEST.in - see https://github.com/pharmbio/sciluigi/pull/35
+  # Manually including a copy.
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
I've also seen this done as `{{ RECIPE_DIR }}/LICENSE`. Is one way preferred over the other?